docs/modules/general/pages/architecture-overview/shell.adoc

```diff
@@ -19,35 +19,40 @@ The Shell application is a special case of OneCX application. It is a host for M
 === Client-side workspace construction
 OneCX platform is operating on the idea of workspaces. Each workspace can have many applications registered and can be customized to its users' needs via custom menu configuration, page theming and many more possibilities. To construct a valid workspace, the following operations happen:
 
-1. Shell BFF gathers workspace information. Data required to construct a valid workspace to function on the platform is saved in several databases. These can be reached using SVCs of OneCX core applications. Shell BFF connects with them and gathers all required information in a single response.
-2. Shell UI constructs routing rules for a given workspace based on the data fetched by Shell BFF.
+. **Shell BFF gathers workspace information**. Data required to construct a valid workspace to function on the platform is saved in several databases. These can be reached using SVCs of OneCX core applications. Shell BFF connects with them and gathers all required information in a single response.
```

Comment:
These can be reached using ~~SVCs~~ backend services of OneCX **Core Applications**.